sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py

```diff
@@ -156,6 +166,7 @@ def send(self, request):
         """
         retryable = True
         redirect_settings = self.configure_redirects(request.context.options)
+        self._original_domain = get_domain(request.http_request.url)
```

Comment:
It looks like we're updating the policy instance? This wont be threadsafe?

Reply:
My understanding is for a created pipeline, request.http_request.urls are same no matter in which thread?